server/services/omnichannel-voip/service.ts

```diff
@@ -17,12 +24,75 @@ export class OmnichannelVoipService extends ServiceClass implements IOmnichannel
 
 	private users: UsersRaw;
 
+	private voipRoom: VoipRoomsRaw;
+
 	constructor(db: Db) {
 		super();
 		this.users = new UsersRaw(db.collection('users'));
+		this.voipRoom = new VoipRoomsRaw(db.collection('rocketchat_room'));
 		this.logger = new Logger('OmnichannelVoipService');
 	}
 
+	private normalizeAgent(agentId: string): ILivechatAgent | { hiddenInfo: true } | undefined {
+		if (!agentId) {
```

Comment:
@pierre-lehnen-rc I have fixed this. But I would like to mention that there is a semantic ambiguity here. Livechat is not a Voip. So returning ILivechatAgent introduces some semantic ambiguity. We have a similar case in for visitors. We are using all the APIs of LiveChat Visitor here. We may have to revisit this.

Reply:
I agree that they are agent. So its not a functional ambiguity. Things will work. What I mean by semantic ambiguity is their name.  But as you have pointed out, we will work towards making the agent more generic.